src/main/java/dinkplugin/util/WorldUtils.java

```diff
@@ -18,17 +17,21 @@ public class WorldUtils {
 
     private final Set<WorldType> IGNORED_WORLDS = EnumSet.of(WorldType.PVP_ARENA, WorldType.QUEST_SPEEDRUNNING, WorldType.NOSAVE_MODE, WorldType.TOURNAMENT_WORLD);
 
+    private final Set<Integer> BA_REGIONS = ImmutableSet.of(7508, 7509, 10322);
+    private final Set<Integer> CASTLE_WARS_REGIONS = ImmutableSet.of(9520, 9620);
+    private final Set<Integer> CLAN_WARS_REGIONS = ImmutableSet.of(12621, 12622, 12623, 13130, 13131, 13133, 13134, 13135, 13386, 13387, 13390, 13641, 13642, 13643, 13644, 13645, 13646, 13647, 13899, 13900, 14155, 14156);
+    private final Set<Integer> COX_REGIONS = ImmutableSet.of(12889, 13136, 13137, 13138, 13139, 13140, 13141, 13145, 13393, 13394, 13395, 13396, 13397, 13401);
     private final Set<Integer> LMS_REGIONS = ImmutableSet.of(13658, 13659, 13660, 13914, 13915, 13916, 13918, 13919, 13920, 14174, 14175, 14176, 14430, 14431, 14432);
+    private final Set<Integer> MAGE_TRAIN_REGIONS = ImmutableSet.of(13462, 13463);
     private final Set<Integer> POH_REGIONS = ImmutableSet.of(7257, 7513, 7514, 7769, 7770, 8025, 8026);
-    private final Set<Integer> SOUL_REGIONS = ImmutableSet.of(8493, 8749, 9005);
-    private final int CASTLE_WARS_REGION = 9520;
-
-    @VisibleForTesting
-    public final int CASTLE_WARS_COUNTDOWN = 380;
-    @Varbit
-    private final int CASTLE_WARS_X_OFFSET = 156;
-    @Varbit
-    private final int CASTLE_WARS_Y_OFFSET = 157;
+    private final Set<Integer> SOUL_REGIONS = ImmutableSet.of(8493, 8748, 8749, 9005);
+    private final Set<Integer> TZHAAR_REGIONS = ImmutableSet.of(9551, 9552);
+    private final int BURTHORPE_REGION = 8781;
+    private final int INFERNO_REGION = 9043;
```

Comment:
I think Inferno and Jad should probably be removed from this, while they do fit the blanket statement of Safe, as you don't lose anything, people would like to know if you died to jad or inferno, since you actually lose out on something.

Reply:
Fixed in https://github.com/pajlads/DinkPlugin/pull/194/commits/941eec62ffc636dcca62df60d7a96c6f5bb65071